Sources/VLCAppDelegate.m

```diff
@@ -283,36 +277,6 @@ - (void)application:(UIApplication *)application performActionForShortcutItem:(U
     //TODO: shortcutItem should be implemented
 }
 
-#pragma mark - media discovering
-
-- (void)mediaFileAdded:(NSString *)fileName loading:(BOOL)isLoading
-{
-    if (!isLoading) {
-        MLMediaLibrary *sharedLibrary = [MLMediaLibrary sharedMediaLibrary];
-        [sharedLibrary addFilePaths:@[fileName]];
-
-        /* exclude media files from backup (QA1719) */
-        NSURL *excludeURL = [NSURL fileURLWithPath:fileName];
```

Comment:
can you make sure that we exclude mediafiles in the future also from backup?

Reply:
By mediafiles do you mean all the users media?

Reply:
yes

Reply:
Updated!